@plotly/dash-generator-test-component-typescript/src/props.ts

```diff
@@ -32,7 +32,7 @@ export type TypescriptComponentProps = {
     array_elements?: JSX.Element[];
 
     string_default?: string;
-    number_default?: string;
+    number_default?: number;
```

Comment:
this seems kind of important :-)

Reply:
I changed this to get tests to pass, but @T4rk1n can you confirm if this is OK?

Reply:
Yes that is the good type.